### PR TITLE
Tile liang.fill op that are not part of any fusion group

### PR DESCRIPTION
**Pull request by ThomasRaoux**

### CODE REVIEW

File: iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp [hunk 1]
```diff
@@ -1017,6 +1017,27 @@ static unsigned makeElementwiseOpsRootOps(FuncOp funcOp, unsigned numRoots) {
   return numRoots;
 }
 
+/// Sets fill op as root op if they are not part of any fusion groups yet.
+static unsigned makeFillOpsRootOps(FuncOp funcOp, unsigned numRoots) {
+  MLIRContext *context = funcOp.getContext();
+  OpBuilder builder(context);
+  for (Block &block : funcOp) {
+    auto linalgOps = block.getOps<linalg::LinalgOp>();
+    for (linalg::LinalgOp linalgOp : llvm::reverse(linalgOps)) {
```
Comment (MaheshRavishankar) [inline iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp:1026]: Nit: Doesnt need to be `reverse`

**Review by MaheshRavishankar — APPROVED** [review]
Awesome! This is what I had in mind!

File: iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp (hunk 1 above)
Comment (ThomasRaoux) [reply iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp:1026]: I guess the order doesn't matter, let me simplify that.

Comment (ThomasRaoux) [reply iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp]: simplified this function a bit

**Pull request closed without merging**